Library/Formula/ipe.rb

```diff
@@ -38,7 +38,7 @@ class Ipe < Formula
 
   def install
     cd 'src' do
-      system "make", "IPEPREFIX=#{prefix}"
+      system "make", "IPEPREFIX=#{HOMEBREW_PREFIX}"
```

Comment:
FWIW, fixed this bug for you the other day: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/ipe.rb#L47-L48